src/makeRobot.js

```diff
@@ -38,7 +38,66 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'wheels': wheels,
+    'version': version,
```

Comment:
Якщо ключ дорівнює іменні змінної можна використати скорочений запис

```suggestion
    name,
    wheels,
    version,
```